src/lib.rs

```diff
@@ -13,3 +13,12 @@ type Global = core::hub::Global<core::hub::IdentityManagerFactory>;
 lazy_static::lazy_static! {
     static ref GLOBAL: Arc<Global> = Arc::new(Global::new("wgpu", core::hub::IdentityManagerFactory));
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn wgpu_get_version() -> *const std::os::raw::c_char {
```

Comment:
should this be a number instead?

Reply:
You mean pack three uint8's (major, minor, patch) into an uint32? That could work. Would you prefer that?

Reply:
Yeah, that would allow to compare it with values like 0x501. I think I would prefer this.
@grovesNL do you have a preference?

Reply:
is this still on your radar?

Reply:
You mean @grovesNL  or me?

Reply:
I mean @almarklein  - to change this to u32

Reply:
Ah, I thought we were waiting for Josh' answer :) I'm on it.

Reply:
Done.